### PR TITLE
Add support for Eclipse Testnet(new)

### PR DESCRIPTION
**Pull request by margin-hub**
Please let me know if you need more info. (New Eclipse Testnet)

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -201,6 +201,23 @@
...
+            "ID": "ECLIPSET",
+            "NAME": "Eclipse Testnet",
+            "TOKEN": "ECLP",
+            "RPC": "https://subnets.avax.network/eclipsecha/testnet/rpc",
+            "CHAINID": 555666,
+            "EXPLORER": "https://subnets-test.avax.network/eclipsecha",
+            "IMAGE": "https://raw.githubusercontent.com/Thor-Financial/Eclipse-assets/main/eclipse-logo.png",
+            "MAX_PRIORITY_FEE": "300000000000",
+            "MAX_FEE": "100000000000",
+            "DRIP_AMOUNT": 1000000000,
```
Comment (rajranjan0608) [inline config.json]: Drip amount is in base units. So above would mean `1000000000 ECLP`. Is that expected?

File: config.json [hunk 2]
```diff
@@ -201,6 +201,23 @@
...
+        },
+        {
+            "ID": "ECLIPSET",
+            "NAME": "Eclipse Testnet",
+            "TOKEN": "ECLP",
+            "RPC": "https://subnets.avax.network/eclipsecha/testnet/rpc",
+            "CHAINID": 555666,
+            "EXPLORER": "https://subnets-test.avax.network/eclipsecha",
+            "IMAGE": "https://raw.githubusercontent.com/Thor-Financial/Eclipse-assets/main/eclipse-logo.png",
+            "MAX_PRIORITY_FEE": "300000000000",
```
Comment (rajranjan0608) [inline config.json]: Priority fee (300 gwei) shouldn't be more than max fee (100 gwei).

**Review by rajranjan0608 — CHANGES_REQUESTED** [review]
Requested a few changes. Feel free to take a look at existing examples: https://github.com/ava-labs/avalanche-faucet/blob/main/config.json#L41-L55

File: config.json (hunk 1 above)
Comment (margin-hub) [reply config.json]: https://docs.avax.network/build/subnet/utility/avalanche-subnet-faucet
Oh, i supposed this amount is in gwei by referring to the above link, so I meant 0.1 ECLP.
0.1ECLP=100000000
Please correct me if I am wrong, I will modify it then.

File: config.json (hunk 2 above)
Comment (margin-hub) [reply config.json]: OKay, i will change that value to 100gwei.

File: config.json (hunk 1 above)
Comment (rajranjan0608) [reply config.json]: Yeah, the docs might be a little outdated. If you want 0.1 ECLP drip, then use 0.1

PS: Not necessary, but I would suggest keeping the drip amount integer.

Comment (margin-hub) [reply config.json]: Thanks for your info, i set it as 1 then.

File: config.json (hunk 2 above)
Comment (rajranjan0608) [reply config.json]: 10 gwei should be sufficient here (that's what we use on C-Chain).

Comment (margin-hub) [reply config.json]: Ah okay, sure will do.

**Comment by margin-hub** [comment]
@rajranjan0608 Please have a look again, i modified 2 param values you mentioned. thanks.

**Review by rajranjan0608 — APPROVED** [review]
Looks good! Thanks for the changes @happyleow

**Comment by rajranjan0608** [comment]
@happyleow Faucet is now live at https://core.app/tools/testnet-faucet?subnet=eclipset&token=eclipset 🚀 

Feel free to recharge the faucet address with some ECLPs

```
0x2352D20fC81225c8ECD8f6FaA1B37F24FEd450c9
```

**Comment by margin-hub** [comment]
@rajranjan0608 Sure, thank you for your support!

**Pull request merged**